CalPatRec/src/DeltaFinder_module.cc

```diff
@@ -58,8 +68,8 @@ namespace mu2e {
       fhicl::Atom<int>             debugLevel             {Name("debugLevel"        )    , Comment("debug level"                ) };
       fhicl::Atom<int>             diagLevel              {Name("diagLevel"         )    , Comment("diag level"                 ) };
       fhicl::Atom<int>             printErrors            {Name("printErrors"       )    , Comment("print errors"               ) };
-      fhicl::Atom<int>             writeFilteredComboHits {Name("writeFilteredComboHits"), Comment("1: write filtered CH coll"  ) };
-      fhicl::Atom<int>             writeStrawHitFlags     {Name("writeStrawHitFlags")    , Comment("1: write SH flag coll"      ) };
+      fhicl::Atom<int>             writeFilteredComboHits {Name("writeFilteredComboHits"), Comment("0: write all CH, 1: write filtered CH") };
```

Comment:
These integers are behaving as booleans, why not define them as such?

Reply:
There is value in declaring variable types explicitly to their purpose.  Here the purpose is to be a boolean, so they should be declared as such.  this has the advantages of creating a standard interface for all Mu2e modules, and allows common-language configuration in the fcl file: ie "true" and "false" vs 1 and 0, which are ambiguous and could mean anything.

Reply:
The issue of correctly typing parameters is outside the scope of this PR.  I will approve and create an issue for this.